engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProviderSingleton.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.openvrprovider;
+
+/**
+ * Created by john on 1/9/17.
+ */
+public class OpenVRProviderSingleton {
+    public static OpenVRProvider vrProvider = new OpenVRProvider();
+}
```

Comment:
P.S. some interesting consideration on this deceptively simple pattern can be found here:

http://www.javaworld.com/article/2073352/core-java/simply-singleton.html